docs/UserGuide.md

```diff
@@ -41,146 +37,167 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
     The index **must be a positive integer** 1, 2, 3, ...
 > * At least one of the optional fields must be provided.
 > * Existing values will be updated to the input values.
-> * When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-> * You can remove all the person's tags by typing `t/` without specifying any tags after it. 
+> * When editing tags, the existing tags of the task will be removed i.e adding of tags is not cumulative.
+> * You can remove all the task's tags by typing `#` without specifying any tags after it. 
 
 Examples:
 
-* `edit 1 p/91234567 e/johndoe@yahoo.com`<br>
-  Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@yahoo.com` respectively.
+* `edit 1 t/9pm p/high #CS1010`<br>
+  Edits the deadline, priority and hashtag of the 1st task to be `9pm` , `high` and `#CS1010` respectively.
 
-* `edit 2 Betsy Crower t/`<br>
-  Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+* `edit 2 t/9pm p/high `<br>
+  Edits the name of the 2nd task's deadline to `9pm` and priority to `high` and clears all existing tags.
 
-### 2.5. Finding all persons containing any keyword in their name: `find`
+### Finding all tasks containing any keyword in their name: `find`
 
-Finds persons whose names contain any of the given keywords.<br>
+Finds tasks whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * The search is case in-sensitive. e.g `cs1231` will  match `CS1231`
+> * Only the task name is searched.
+> * Only full words will be matched e.g. `CS2103T` will not match `CS2103`
+> * Tasks matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `OP` will match `OP1 OP`
 
 Examples:
 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
-
-### 2.6. Deleting a person : `delete`
-
-Deletes the specified person from the address book. Irreversible.<br>
+* `find OP` <br>
+  Returns `OP1`
+  
+* `find OP1 OP2 Lab1`<br>
+  Returns Any task having names `Lab1`, `OP1`, or `OP2` 
+ 
+### Deleting specified items from the task manager: `delete`
+Deletes the specified task from the task manager.<br>
 Format: `delete INDEX`
 
-> Deletes the person at the specified `INDEX`. <br>
+> Deletes the task at the specified `INDEX`. <br>
 > The index refers to the index number shown in the most recent listing.<br>
 > The index **must be a positive integer** 1, 2, 3, ...
 
 Examples:
 
 * `list`<br>
```

Comment:
I see that you mentioned `list` here, but I don't find this `list` command anywhere in this user guide.